backends/libParanumal/ceed-libparanumal-qfunction.c

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017-2018, Lawrence Livermore National Security, LLC.
+// Produced at the Lawrence Livermore National Laboratory. LLNL-CODE-734707.
+// All Rights reserved. See files LICENSE and NOTICE for details.
+//
+// This file is part of CEED, a collection of benchmarks, miniapps, software
+// libraries and APIs for efficient high-order finite element and spectral
+// element discretizations for exascale applications. For more information and
+// source code availability see http://github.com/ceed.
+//
+// The CEED research is supported by the Exascale Computing Project 17-SC-20-SC,
+// a collaborative effort of two U.S. Department of Energy organizations (Office
+// of Science and the National Nuclear Security Administration) responsible for
+// the planning and preparation of a capable exascale ecosystem, including
+// software, applications, hardware, advanced system engineering and early
+// testbed platforms, in support of the nation's exascale computing imperative.
+#include "ceed-libparanumal.h"
```

Comment:
This backend does not impliment QFunctionApply. If this is the case, then it should have a dummy implimentation that returns an error message to that effect:

```
// *****************************************************************************
// * CeedBasisCreateH1_Occa
// *****************************************************************************
int CeedBasisCreateH1_Occa(CeedElemTopology topo, CeedInt dim,
                          CeedInt ndof, CeedInt nqpts,
                          const CeedScalar *interp,
                          const CeedScalar *grad,
                          const CeedScalar *qref,
                          const CeedScalar *qweight,
                          CeedBasis basis) {
  return CeedError(basis->ceed, 1, "Backend does not implement non-tensor bases");
}
```

Reply:
I think in terms of usability, it would be ideal to use the default OCCA implimentation unless there is a libParanumal implimentation so that the user has something (with OperatorApply too?). Perhaps some backends should have READMEs for information like that?